src/onpage.js

```diff
@@ -18,9 +19,8 @@
 		scriptTag.src = 'SCRIPT_URL_HERE';
 		firstScript = doc.getElementsByTagName(scriptStr)[0];
 		firstScript.parentNode.insertBefore(scriptTag, firstScript);
-
-		root[branchStr] = branchSdk;
 	}
+	else { console.log("Branch is already embedded" ); }
```

Comment:
maybe we can verify it's branch with like `else if (!(typeof root[branchStr] == 'object' && root[branchStr].init && root[branchStr].data && root[branchStr].link)) { console.log('Conflict: you already have another global varibale named branch') }`